### DISCUSSION TITLE
Content must be served over HTTPS error for requests to minio in my self hosting configuration

### DISCUSSION THREAD

**Discussion by MSVO** [outline/outline#4976] [Self Hosting]
Hello Outline Community!

I self hosted outline following the guide here for docker-compose https://wiki.generaloutline.com/s/hosting/doc/docker-7pfeLP5a8t  But I removed the https-portal service block from the docker-compose in favor of my own nginx reverse proxy and certificate configuration in the host machine. I am able to edit pages in the publicly accessible self hosted instance through browser. However when I try to import a zip through outline settings, the upload fails and in the Firefox console I could see an error like this: 

`Mixed Content: The page at 'https://outline.mydomain.com/settings/import' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://storage:9000/outline_bucket'. This request has been blocked; the content must be served over HTTPS.`

I have pointed to `http://storage:9000` in docker.env as I believe it is the url that docker can resolve to the `storage` service which is present in docker-compose. I don't know how the request got carried over to browser. Please help me debug this issue.

**Comment by WarlockFish** [comment]
This is caused by firefox security， There are confusing links between http and https in your page 。

https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content


 You can temporarily turn off security for your website   or  add ssl for storage “https://storage:9000”

![图片](https://user-images.githubusercontent.com/30708115/230519643-7e0c9939-5246-4ffa-bab6-31db409a7ead.png)
